department-of-veterans-affairs/vets-website
# Income and asset 81952

## THIS FORM IS DISABLED IN PRODUCTION

## Summary
Income and Asset Statement Form (21P-0969)

## Issue
[Generate new vets-website application#81952](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81952)

### Related Issues
n/a

### Epic
[Digitize Income and Asset Statement Form 21P-0969](https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/gh/department-of-veterans-affairs/va.gov-team/80925)

### Screenshots
n/a

## How to run in local environment
1. Check out this branch locally in 'vets-website'
2.  Checkout this [branch](https://github.com/department-of-veterans-affairs/content-build/pull/2075) locally in 'content-build'
3. Run 'vets-api' in your local environment
4. Run the 'vets-website' app with `yarn watch --env entry=static-pages,auth,login-page,profile,dashboard,income-and-asset-statement`

## How to toggle feature flag in local environment
1. Go to [income_and_assets_form_enabled](http://localhost:3000/flipper/features/income_and_assets_form_enabled)

## How to verify
1. Go to `http://localhost:3001/income-and-asset-statement-form-21p-0969` in your browser
2. Verify the app loads in the browser
3. Verify that form progress is saved to the api and in the redux store
4. Verify that the form can be submitted

## Testing

## What areas of the site does it impact?
Income and Asset Statement Application

### Quality Assurance & Testing

### Error Handling

### Authentication
